packages/api/cms-api/src/page-tree/page-tree.module.ts

```diff
@@ -65,6 +67,13 @@ export class PageTreeModule {
 
         const documentSubscriber = DocumentSubscriberFactory.create({ Documents });
 
+        const sitePreviewProvider: Provider<SitePreviewConfig> = {
```

Comment:
I'd call it `sitePreviewConfigProvider`

Reply:
82325bcec467fb079797df716322fc5cb71f8c2a